docs/scenarios/report/README.md

```diff
@@ -0,0 +1,41 @@
+## Generating pre-migration report with CPMA
+
+---
+
+### 1. Prerequisites
+
+Prior to working with CPMA you need to deploy an OCP3 cluster(OCP 3.7, 3.9, 3.10, 3.11 should be supported).
```

Comment:
nit

Change 'deploy an OCP3 cluster' to 'deploy an OCP 3.7+ cluster'
Change 'should be supported' to 'will be supported'

Reply:
Or maybe "are supported" ?